CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+### 2.32.0
+This release includes:
+* Fluent Bit [1.9.10](https://github.com/fluent/fluent-bit/tree/v1.9.10)
+* Amazon CloudWatch Logs for Fluent Bit 1.9.4
+* Amazon Kinesis Streams for Fluent Bit 1.10.2
+* Amazon Kinesis Firehose for Fluent Bit 1.7.2
+
+Compared to `2.31.12.20230911` this release adds:
```

Comment:
These notes are not updated. Resolve before merge

Reply:
Also the release note should be Compared to `2.31.12`, the date stamped re-builds for linux only don't count as a full release.

Reply:
Sure! Updated.